code/style.css

```diff
@@ -1,7 +1,465 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+/* 
+Project requirements
 
-.active + .answer {
-  /* some styling here */
+🔵  Blue Level (Minimum Requirements)
+
+- Style your page
+- Site should have a header with responsive image/video
+- Site should have a signup form with at least three different input types
+- Site should have an FAQ accordion, built with JavaScript 
+
+🔴  Red Level (Intermediary Goals)
+
+- Try out some other form elements or input types. [Here](https://www.w3schools.com/html/html_form_elements.asp), you can read about what other form elements there are
+- Add an icon before the title in the accordion indicates whether the section is expanded or collapsed. It should update when the section expands or collapses.
+- Add some [validation](https://technigo.wistia.com/medias/bvq814g0bk) to the form
+- Use CSS selectors to make every other row in the accordion have another background color, or add a focus effect on the input tags. [Here](https://www.w3schools.com/cssref/css_selectors.asp) you can check what selectors to use.
+- On hover, change the mouse cursor to something other than the default with help from 👉 [MDN Web docs: cursor](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor)
+
+⚫  Black Level (Advanced Goals)
+
+- Add some animations to your project. For example, a slide-down effect when the accordion expands and collapses.
+- One of the harder things to do in CSS is to style checkboxes and radio buttons. Try to create custom [checkboxes](https://www.notion.so/Create-and-style-checkboxes-77dfbec5fa8346978f02db9934f447e8) and [radio buttons](https://www.notion.so/Create-and-style-radio-buttons-0739d9f6b7824c748f78f20e231bdb14).
+- Add a header and/or a button on top of your video, just like Confetti, by using `position: absolute;` and `z-index` to get everything in the right place.
+- Read up on the [BEM](http://getbem.com/introduction/) methodology and use it for the project
+*/
```

Comment:
Nice to have easy access to this info while working with the project. An suggestion is to put this information in the README right before hand-in to slim the code in css a bit.